File: src/main/java/me/dolphago/GithubFollowerTrackerApplication.java
package me.dolphago;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.openfeign.EnableFeignClients;
import org.springframework.data.jpa.repository.config.EnableJpaAuditing;

@EnableJpaAuditing
@EnableFeignClients
@SpringBootApplication
public class GithubFollowerTrackerApplication {
    public static void main(String[] args) {
        SpringApplication.run(GithubFollowerTrackerApplication.class, args);
    }
}

File: src/main/kotlin/me/dolphago/GithubFollowerTrackerApplication.kt
package me.dolphago

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.cloud.openfeign.EnableFeignClients
import org.springframework.data.jpa.repository.config.EnableJpaAuditing

@EnableJpaAuditing
@EnableFeignClients
@SpringBootApplication
class GithubFollowerTrackerApplication

fun main(args: Array<String>) {
    SpringApplication.run(GithubFollowerTrackerApplication::class.java, *args)
}
